src/cls/IPM/Lifecycle/Base.cls

```diff
@@ -255,48 +255,46 @@ Method CheckBeforeClean(
 	ByRef pParams,
 	Output pSkip As %Boolean = 0) As %Status
 {
-    set tSC = $$$OK
-    try {
-        set tVerbose = $get(pParams("Verbose"))
-        set tLevel = $get(pParams("Clean","Level"),0)
-        set tRecurse = $get(pParams("Clean","Recurse"),1)
-
-        if (..Module.GlobalScope && '$get(pParams("Clean","GlobalScope"))) {
-            do ..Log("Clean SKIPPED - module has global scope.")
-            set pSkip = 1
-            quit
-        }
-
-        if '$get(pParams("Clean","Force")) {
-            // Check to see if anything depends on this module and return an error status if it does.
-            set tSC = ##class(%IPM.Utils.Module).GetDependentsList(.tList,,..Module.Name)
-            if $$$ISERR(tSC) {
-                quit
-            }
-
-            set tModList = ""
-            for i=1:1:tList.Count() {
-                set tName = tList.GetAt(i).Name
-                if '$data(pParams("Clean","Cycle",tName)) {
-                    set tModList = tModList_$listbuild(tName)
-                }
-            }
+  set tSC = $$$OK
```

Comment:
@isc-dchui Looks like the local folder settings.json did not get used for formatting rules for some reason? That has tabSize set to 4 which would make the diff without the format differences. Could you adjust the tab size back to 4?

Reply:
Oh yeah probably should remove it or keep it in sync with the vs code settings file